src/converters/fromZigbee.ts

```diff
@@ -1424,6 +1425,22 @@ const converters1 = {
                 payload.action_transition_time = msg.data.transtime / 100;
             }
 
+            if (options.simulated_color_temperature) {
+                const deltaOpts = options.simulated_color_temperature_delta ?? 500;
```

Comment:
I assume the delta is in Kelvin, I propose to use mireds here as that is what the ZCL uses.